server/lib/disaster_dashboard.py

```diff
@@ -21,7 +21,7 @@
 EVENT_TYPES = [
     "FireEvent", "WildlandFireEvent", "WildfireEvent", "CycloneEvent",
     "HurricaneTyphoonEvent", "HurricaneEvent", "TornadoEvent", "FloodEvent",
-    "DroughtEvent", "WetBulbTemperatureEvent"
+    "DroughtEvent", "WetBulbTemperatureEvent", "ColdEvent", "HeatEvent"
```

Comment:
I believe these should be "HeatTemperatureEvent" and "ColdTemperatureEvent", per latest email from  @jehangiramjad 

https://github.com/datacommonsorg/schema/blob/main/core/natural_phenomena.mcf#L425-L435

Reply:
Yes please :)

Reply:
Also @ajaits as FYI

Reply:
PR to update this: https://github.com/datacommonsorg/website/pull/2098